deploy/app/bin/deploy-staging.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-ansible-playbook ./deploy-app.yml -i ./hosts/staging -u root --extra-vars "env=staging" "$@"
+ansible-playbook ./deploy-app.yml -i ./hosts/staging --extra-vars "env=staging" "$@"
```

Comment:
we might need to use `./deploy-app.sh` here

Reply:
@anorsich How do you propose to use `deploy-app.sh`? Do you mean, that we can delete bash file `deploy-staging.sh`?